src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -204,12 +204,6 @@ void GcsClient::Disconnect() {
     return;
   }
   is_connected_ = false;
-  periodical_runner_.reset();
-  gcs_subscriber_.reset();
-  if (redis_client_) {
-    redis_client_->Disconnect();
```

Comment:
Is it an issue that we're no longer disconnecting the Redis client? Doesn't seem to do much more than flip an `is_connected_` flag, which itself doesn't seem to be used for much.

Reply:
We reset gcs client after joining a thread

Reply:
Yep I know, I was referring to `RedisClient::Disconnect()`, which AFAICT is no longer called. Seems to be fine since it doesn't do much, but just wanted to call that out.

https://github.com/ray-project/ray/blob/master/src/ray/gcs/redis_client.cc#L193

Reply:
Ah yeah. When I looked at that code, it didn't seem to be anything, so I think it is okay (`is_connected_` that's flipped is not used iirc)